### ISSUE TITLE
swipe keyboard:  'modules' does not exist in type 'KeyboardOptions'

### ISSUE THREAD

**Issue by JeffWinder** [hodgef/simple-keyboard#421]
**Simple-keyboard version**
+ swipe-keyboard@2.0.84
+ simple-keyboard@2.27.78

**Describe the bug**
Importing in Angular 8 project:
```
import Keyboard from "simple-keyboard";
import swipe from 'swipe-keyboard';
```
Declaring class var:
`keyboard: Keyboard;`

in ngOnInit:
```
  this.keyboard = new Keyboard({
      newLineOnEnter: true,
      onChange: input => this.onChange(input),
      onKeyPress: button => this.onKeyPress(button),
      useMouseEvents: true,
      modules: [swipe]
    });
```
This gives me the error:
_Argument of type '{ newLineOnEnter: true; onChange: (input: string) => void; onKeyPress: (button: string) => void; useMouseEvents: true; modules: any[]; }' is not assignable to parameter of type 'KeyboardOptions'.
  Object literal may only specify known properties, and 'modules' does not exist in type 'KeyboardOptions'._

**Comment by hodgef** [comment]
Hello @JeffWinder,

Sorry about the missing type! It should be now fixed with the newly released version 2.28.6.

Regards,
Francisco Hodge

**Comment by JeffWinder** [comment]
confirming the error is gone in 2.28.6., thanks!
